go/client/ui.go

```diff
@@ -521,6 +521,47 @@ func (p ProveUI) DisplayRecheckWarning(_ context.Context, arg keybase1.DisplayRe
 
 //============================================================
 
+type UpdateUI struct {
+	terminal libkb.TerminalUI
+	noPrompt bool
+}
+
+func (u UpdateUI) UpdatePrompt(_ context.Context, arg keybase1.UpdatePromptArg) (res keybase1.UpdatePromptRes, err error) {
+	if u.noPrompt {
+		res.DoInstall = true
+		return res, err
+	}
+	u.terminal.Printf("There is a new update available (version %s)\n", arg.Version)
+	if len(arg.Desc) != 0 {
+		u.terminal.Printf("  %s\n", arg.Desc)
+	}
+	prompt := "Install update?"
+	res.DoInstall, err = u.terminal.PromptYesNo(PromptDescriptorUpdateDo, prompt, libkb.PromptDefaultYes)
+	if err != nil {
+		return res, err
+	}
+	if res.DoInstall {
```

Comment:
We don't get to here if already updating, since there's no prompt requested.

I can go either way on always asking.  It seemed a little bit weird to say no to this one, but to want to blindly install in the future.